js/NavMainComponent.js

```diff
@@ -87,16 +87,23 @@ NavMainComponent.prototype.init = function () {
     // Open quaternary nav on tertiary nav item click
     component.$tertiaryNavLinks.on('click', function (event) {
         var $self = $(this),
-            href = $self.attr('href');
+            target;
 
-        if (href.indexOf('#') !== -1) {
+        if ($self.attr('href')) {
+            target = $self.attr('href');
+        }
+        else if ($self.attr('data-target')) {
+            target = $self.attr('data-target');
+        }
+
+        if (target.indexOf('#') !== -1) {
```

Comment:
Just a point of discussion here not something that necessarily needs actioning. We still have the ability here to call `.indexOf()` on `undefined` which will throw the same error this ticket is fixing. 

As I see it we have 2 options: 

We could make sure that this code could never throw an error and return inside this function early if `target` is `undefined`. This means that if I implement the nav menu markup incorrectly and omit those attributes, I'll never know it doesn't work until a manually test it. We would effectively be swallowing that implementation error for the sake of it not making it to production.

Or we leave this as is, and put a meaningful error in this block if `target` is `undefined`, we could throw an error a long the lines of `A nav link target requires a href or data-target attribute`.

I think door number 2 would be the friendlier approach for product teams.